openedx/tests/xblock_integration/test_recommender.py

```diff
@@ -250,7 +250,7 @@ def generate_edit_resource(self, resource_id):
         """
         resource = {"id": resource_id}
         edited_recommendations = {
-            key: value + " edited" for key, value in self.test_recommendations[self.resource_id].iteritems()
+            key: value + "edited" for key, value in self.test_recommendations[self.resource_id].iteritems()
```

Comment:
To close the loop: It was being used to denote that a URL had been edited in some way, and since the recommender update removed the ability to add invalid URLs as locations, the space needed to be removed.